src/NodaTime.Demo/OffsetDateTimeDemo.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright 2019 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using NUnit.Framework;
+
+namespace NodaTime.Demo
+{    
+    public class OffsetDateTimeDemo
+    {
+        [Test]
+        public void Construction()
+        {
+            LocalDateTime localDateTime = new LocalDateTime(1985, 10, 26, 1, 18);
+            Offset offset = Offset.FromHours(-5);
+            OffsetDateTime offsetDateTime = Snippet.For(new OffsetDateTime(localDateTime, offset));
+
+            Assert.AreEqual(localDateTime, offsetDateTime.LocalDateTime);
+            Assert.AreEqual(offset, offsetDateTime.Offset);
+        }
+
+        [Test]
+        public void WithCalendar()
+        {
+            LocalDateTime localDateTime = new LocalDateTime(1985, 10, 26, 1, 18, CalendarSystem.Gregorian);
```

Comment:
It's probably worth using calendar systems where the date actually changes - ISO and Gregorian are effectively synonyms. I'd probably use ISO to start, then convert to Julian.